app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,31 @@
+module SessionsHelper
```

Comment:
plase make sure to move all your authentication methods to `app/controllers/application_controller.rb` so they can be available to all controllers

Reply:
We included the module SessionsHelper on the ApplicationController
```suggestion
class ApplicationController < ActionController::Base
  include SessionsHelper
end
```